lib/matplotlib/tests/test_widgets.py

```diff
@@ -14,16 +14,21 @@
                                         mock_event, noop)
 
 import numpy as np
-from numpy.testing import assert_allclose
 
 import pytest
 
-
```

Comment:
Thanks for rebasing this!

If you add this blank line back, the now failing tests should pass.